.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+version: 2.1
+
+commands:
+  auth-regress:
+    description: "run one auth regression context"
+    parameters:
+      skip:
+        type: string
+        default: ""
+      context:
+        type: string
+        default: ""
+    steps:
+      - run: |
+          cd regression-tests
+          [ -e ./vars ] && . ./vars
+          rm -rf tests/*/skip
+          for t in << parameters.skip >>
+          do
+            touch tests/$t/skip
+          done
+          ./start-test-stop 5300 << parameters.context >>
+
+jobs:
+  build:
+    docker:
+      - image: debian:stretch
+      - image: mcr.microsoft.com/mssql/server:2019-CTP2.2-ubuntu
+        environment:
+          - ACCEPT_EULA: Y
+          - SA_PASSWORD: 'SAsa12%%'
+
+    steps:
+      - checkout
+
+      - run:
+          name: install dependencies
+          command: ./build-scripts/circleci.sh debian-stretch-deps
+
+      - run:
+          name: autoconf
+          command: autoreconf -vfi
+
+      - run:
+          name: configure
+          command: ./configure --disable-lua-records --with-modules='bind gmysql godbc random'
```

Comment:
Out of curiosity, why do we build with `gmysql` and `random` here? I would have expected `gsqlite3` instead.

Reply:
No good reason at all, just what I ended up accidentally with after experimenting for a while.